### PR TITLE
Started initial work on AOP with User ID Map.

### PR DESCRIPTION
**Pull request by arekmano**
- Created UserAspect, which has an 'around' for the find method.
- Added Unit tests for UserIdMap and UserMapper.
- Linter Autoformatted some file spacings.

### CODE REVIEW

File: app/mapper/UserMapper.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
 import UserIdMap
 import UnitOfWork
-
+import pdb
```
Comment (lancelafontaine) [inline app/mapper/UserMapper.py]: pdb is not currently being used?

File: app/mapper/UserMapper_test.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
+from app.mapper import UserMapper
+from app.mapper import UserIdMap
+from app.core.user import User
+from app.TDG import UserTDG
+import pytest
+import pdb
```
Comment (lancelafontaine) [inline app/mapper/UserMapper_test.py]: Also here, no being used?

File: app/mapper/UserMapper.py (hunk 1 above)
Comment (arekmano) [inline app/mapper/UserMapper.py]: fixed

File: app/mapper/UserMapper_test.py (hunk 2 above)
Comment (arekmano) [inline app/mapper/UserMapper_test.py]: fixed

**Pull request merged**